float64 probability 
float64 x 
float64 y 
float64 w 
float64 h
int32 class_id
